PWGHF/Utils/utilsAnalysis.h

```diff
@@ -36,6 +36,27 @@ int findBin(T1 const& binsPt, T2 value)
   }
   return std::distance(binsPt->begin(), std::upper_bound(binsPt->begin(), binsPt->end(), value)) - 1;
 }
+
+/// Single-track cut on DCAxy
+/// \param trackPt is the prong pT
+/// \param dcaXY is the prong dcaXY
+/// \return true if track passes all cuts
+template <typename T1, typename T2>
+bool isSelectedTrackDcaXY(T1 const& binsPtTrack, T2 const& cutsSingleTrack, const float& trackPt, const float& dcaXY)
```

Comment:
Do not pass simple types by reference.
```suggestion
bool isSelectedTrackDcaXY(T1 const& binsPt, T2 const& cuts, const float pt, const float dcaXY)
```